serverless/src/util/aws/getSearchExportBucket.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Returns the search export bucket name from the env variable or the hard-coded offline value
+ * @returns {string} the bucket name
+ */
+export const getSearchExportBucket = () => {
+  let bucket = process.env.searchExportBucket
+
+  if (process.env.IS_OFFLINE) {
+    // If we are running locally offline, this is the queueUrl
```

Comment:
This should say "... this is the bucket name"

Also, I dont know if this is too nitpicky but I think its odd to structure commets in an first person voice. I prefer if we stick with something like "If running locally, set a default bucket" 🤷

Reply:
no problem at all.  This has been fixed with https://github.com/nasa/earthdata-search/pull/1630/commits/2152c71e014bbc4247b32b73b14efa0193285f58